src/SFA.DAS.AssessorService.Application/Handlers/Staff/GetLearnerDetailHandler.cs

```diff
@@ -86,6 +86,7 @@ public class GetLearnerDetailHandler : IRequestHandler<GetLearnerDetailRequest,
                 LearnStartDate = certificateData.LearningStartDate.HasValue ? certificateData.LearningStartDate : learner?.LearnStartDate,
```

Comment:
Can we rework the handler a little bit while we are here?
Take out the `var standard = await _standardRepository.GetStandardCollationByStandardId(request.StdCode);`

Replace it with `GetStandardVersionByLarsCode(code,version)` then it's another collation call taken away. 

It would be nice if this had some unit tests but looking at it, there aren't any for the handlers as it stands currently, which is a shame. Given we want to get on with the admin stuff can you add a tech debt ticket to SV-363 to add Unit tests to the GetLearnerDetailsHandler?